exercises/calc/calc.py

```diff
@@ -63,35 +70,35 @@ def parse(s, i, ts):
 
 def main():
 
-    p = make_seq(num_parser, make_seq(op_parser,num_parser))
+    p = make_seq(num_parser, make_seq(op_parser,num_parser))  # Create parser for number and operator sequence
     s = ''
     iface = 'eth0'
 
     while True:
-        s = input('> ')
+        s = input('> ')  
         if s == "quit":
             break
         print(s)
         try:
-            i,ts = p(s,0,[])
+            i,ts = p(s,0,[])  
+            # Construct packet using parsed tokens
             pkt = Ether(dst='00:04:00:00:00:00', type=0x1234) / P4calc(op=ts[1].value,
                                               operand_a=int(ts[0].value),
                                               operand_b=int(ts[2].value))
             pkt = pkt/' '
 
-#            pkt.show()
-            resp = srp1(pkt, iface=iface, timeout=1, verbose=False)
+            resp = srp1(pkt, iface=iface, timeout=1, verbose=False)  # Send packet and receive response
             if resp:
                 p4calc=resp[P4calc]
                 if p4calc:
-                    print(p4calc.result)
+                    print(p4calc.result)  # Print the result from the response packet
                 else:
                     print("cannot find P4calc header in the packet")
             else:
                 print("Didn't receive response")
         except Exception as error:
-            print(error)
+            print(error)  # Print any exceptions that occur during parsing or packet handling
 
 
 if __name__ == '__main__':
-    main()
+    main()  # Call the main function
```

Comment:
This comment is unnecessary because it is clear.

Reply:
> This comment is unnecessary because it is clear.
Thank you for the suggestion. I thought it might help someone. But i know keeping the code clean and clutter free is also important

Reply:
@Arnud You can make further changes to this PR by committing to the branch of your fork of the tutorials repo with those changes, and this PR will automatically update as a result.  Were you planning on updating this PR based on the suggested changes in comments?

Reply:
Yes I am

Reply:
> @Arnud You can make further changes to this PR by committing to the branch of your fork of the tutorials repo with those changes, and this PR will automatically update as a result. Were you planning on updating this PR based on the suggested changes in comments?

I have commited the changes. Please review them